design-toolkit/components/tsconfig.json

```diff
@@ -10,6 +10,7 @@
   ],
   "compilerOptions": {
     "jsx": "react-jsx",
+    "lib": ["ESNext", "DOM"],
```

Comment:
This was necessary to be able to use Set methods, but should ultimately be handled in the tsconfig package